src/main/java/com/google/step/youtube/PromoCode.java

```diff
@@ -4,18 +4,20 @@
 import java.util.Date;
 
 /**
- * Value type for a promotional code and information on its source. 
+ * Value type for a promotional code and information on its source.
  */
 @AutoValue
 public abstract class PromoCode {
 
-    public static PromoCode create (String promoCode, String videoId, Date videoUploadDate) {
-        return new AutoValue_PromoCode(promoCode, videoId, videoUploadDate);
+    public static PromoCode create(String promoCode, String videoId, String videoTitle, Date videoUploadDate) {
```

Comment:
Did adding a field not break the PromoCodeServletTest? It runs PromoCode.create(promoCode, videoId, date)